_posts/how-do-we-test-and-document-our-c-api.md

```diff
@@ -0,0 +1,422 @@
+---
+title: How do we Test and Document our C API?
```

Comment:
This title gives very little context to someone that sees it for the first time (who is we, what is our stack in?)

Something more relatable should be:

```suggestion
title: Developing and Testing C libraries from Rust
```